components/automate-cli/pkg/diagnostics/integration/iam_v2.go

```diff
@@ -46,6 +54,16 @@ func CreateIAMV2Diagnostic() diagnostics.Diagnostic {
 		Tags: diagnostics.Tags{"auth", "iam-v2", "skip-for-deep-upgrade"},
 		Generate: func(tstCtx diagnostics.TestContext) error {
 			tstCtx.SetValue("iam-v2-policy-id", save{PolicyID: policyID})
+			err := MustJSONDecodeSuccess(
+				tstCtx.DoLBRequest("/apis/iam/v2beta/roles",
+					lbrequest.WithMethod("POST"),
+					lbrequest.WithJSONStringTemplateBody(roleCreateTemplateStr, struct{}),
```

Comment:
Won't compile, needs to be
```suggestion
					lbrequest.WithJSONStringTemplateBody(roleCreateTemplateStr, struct{}{}),
```